src/custom/components/NotificationBanner/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import styled from 'styled-components/macro'
+import { Colors } from 'theme/styled'
+import { X } from 'react-feather'
+
+export interface BannerProps {
+  children: React.ReactNode | string
+  level: 'info' | 'warning' | 'error'
+  visible: boolean
+}
+
+const Banner = styled.div<{ isActive: any; level: 'info' | 'warning' | 'error' }>`
```

Comment:
`isActive` is a boolean, no?

```suggestion
const Banner = styled.div<{ isActive: boolean; level: 'info' | 'warning' | 'error' }>`
```

Reply:
Also, you could reuse the `level`.

Define a type `level` and  use it on BannerProps and here.

Alternatively, you can use `BannerProps` here:

```suggestion
const Banner = styled.div<Pick<BannerProps, 'visible' | 'level'>>`
```

Of course, you'll need to replace `isActive` with `visible` which is the same already anyway.

Also also :) I suggest renaming `visible` to `isVisible`